.github/workflows/npm-publish.yml

```
@@ -0,0 +1,31 @@
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: 'Node.js Package'
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish-npm:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: 'actions/checkout@v2'
+
```

```suggestion
  publish-npm:
    runs-on: 'ubuntu-latest'
    
    strategy:
      matrix:
        node-version: [14.x]

    steps:
      - uses: 'actions/checkout@v2'

```

I think you need to define `matrix.node-version` if you want to use it in `${{ matrix.node-version }}`. 😉

---

By default `actions/setup-node@v2` will use the latest LTS version of Node.js, and we only want to release once, so it makes sense to only run the action on a single version of `Node.js`, and to don't use `matrix.node-version`.

---

Oh okay, so you don't _need_ to specify a version when using `actions/setup-node@v2` then... 😄 Well I learned something new. 😄

---

Oh my bad, actually I checked again but you were right, excuse me for misleading you.

In fact, I copied the config file from https://github.com/Divlo/create-fullstack-app/ that was also copied from the docs, but I certainly not on purpose deleted these 2 lines :
```yaml
node-version: 14
registry-url: 'https://registry.npmjs.org/'
```

As it is working for `create-fullstack-app`, I thought, it would also working there, but I forgot to copy these 2 lines so if it has not the same configuration he could not work the same ah ah. :laughing: 

But it is better to use only 1 version to release, so I don't think we need to still, declare `strategy.matrix` etc.
What do you think ? @HonkingGoose

---

> Oh my bad, actually I checked again but you were right, excuse me for misleading you.

Oh well, we all make mistakes.. 😄 
So we **do need** to define at least one Node version.

> But it is better to use only 1 version to release, so I don't think we need to still, declare strategy.matrix etc.
What do you think ? @HonkingGoose

I agree, just use 1 version to create the release.
I think if you ran this action with a Node version matrix, you'd probably make multiple releases (1 release for Node v12, 1 release for Node v14 etc.)